README.md

```diff
@@ -75,6 +76,7 @@ with each file. In JSON, the metadata looks something like this:
             "end": "2015-03-20T23:59:59.114Z",
             "where": "webserver02",
```

Comment:
Should include the program that created it with it's version? That might be captured in the data version thou.